Makefile

```diff
@@ -102,21 +102,28 @@ research:
 azure-login:
 	az account set -s $(AZURE_SUBSCRIPTION)
 
+read-keyvault-config:
+	$(eval key_vault_name=$(shell jq -r '.key_vault_name' terraform/workspace_variables/$(APP_ENV).tfvars.json))
```

Comment:
I was wondering why those weren't capitilized to be consistent with the variables defined above?

Reply:
Thanks, it looks better indeed